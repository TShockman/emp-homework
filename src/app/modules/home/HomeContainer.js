import {connect} from 'react-redux';
import Home from './Home';

function mapStateToProps(state) {
  return {}
}

const mapDispatchToProps = {};

export default connect(mapStateToProps, mapDispatchToProps)(Home);
