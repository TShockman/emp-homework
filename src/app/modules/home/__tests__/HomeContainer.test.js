/* eslint-disable import/first */
jest.mock('../Home', () => 'Home');

import React from 'react';
import renderer from 'react-test-renderer';
import {createStore} from 'redux';
import {Provider} from 'react-redux';
import HomeContainer from '../index';

describe('HomeContainer', () => {
  it('Provides Home with correct props', () => {
    const store = createStore(x => x, {});
    const tree = renderer.create(
        <Provider store={store}>
          <HomeContainer/>
        </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});