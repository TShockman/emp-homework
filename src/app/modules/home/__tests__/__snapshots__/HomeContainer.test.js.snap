// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContainer Provides Home with correct props 1`] = `<Home />`;
